README.md

```diff
@@ -80,6 +80,40 @@ is.
 End tags must be omitted when possible and attribute values must not be quoted (if necessary, use
 double quotes).
 
+Place one new line between paragraphs (including list elements). Place two or more new lines before
+headings (e.g., `<h3>`). This doesn't apply when a nested heading follows the parent heading.
```

Comment:
Headings depend on their heading level, but I've probably not always been consistent. `<h2>` gets three newlines, the remainder two. As the size of the document increases it might make sense to adjust this to give even more spacing for the larger sections.

Reply:
Done.